src/components/UniversalNavbarExpanded/UniversalNavbarExpanded.module.scss

```diff
@@ -89,6 +89,26 @@ $laptop-range-small-end: 929px;
   z-index: $z-index-navbar;
 }
 
+:global(.isScrolled) {
+  .laptopAndUpAnimation {
+    background-color: var(--White);
+    color: var(--GrayPrimary--translucent);
+    border-bottom: $navbar-border;
+    transition: 1s ease-in-out;
+  }
+}
+
+:global(.isUnscrolled) {
+  .laptopAndUpAnimation {
+    transition: 1s ease-in-out;
+  }
+}
+
+.laptopAndUpAnimation {
+  border-bottom: none;
+  background-color: transparent;
```

Comment:
minor: I think you can have the `transition: 1s ease-in-out;` rule in this `.laptopAndUpAnimation` just once instead of in two locations above. Not blocking.

Reply:
Good point. I just added the changes.